tekton/images/test-runner/Dockerfile

```diff
@@ -42,6 +42,8 @@ RUN go get -u github.com/golang/dep/cmd/dep
 RUN go get -u github.com/google/licenseclassifier
 RUN go get -u github.com/jstemmer/go-junit-report
 RUN GO111MODULE="on" go get -u github.com/raviqqe/liche
+RUN GO111MODULE="on" go get -u knative.dev/test-infra/tools/githubhelper
```

Comment:
Interesting, I though we were not using `githubhelper` anymore, you may be outdated :sweat_smile: